test/functional/apps/discover/_discover.ts

```diff
@@ -120,6 +120,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
           );
           return actualCount === expectedCount;
         });
+        const prevRowData = await PageObjects.discover.getDocTableField(1);
```

Comment:
Can we maybe make this `let` and move it somewhere on top of the test suite, so it's more clear what it's used for?

Reply:
Open for changes. Why would in this case defining the variable via `let` at the beginning of the test make it more clear?

Reply:
synced offline and decided it's fine this way